temoa/components/limits.py

```
@@ -809,17 +864,23 @@ def LimitEmission_Constraint(M: 'TemoaModel', r, p, e, op):
     return expr
 
 
-def LimitGrowthCapacityConstraint_rule(M: 'TemoaModel', r, p, t, op):
+def LimitGrowthCapacityConstraint_rule(
+    M: 'TemoaModel', r: Region, p: Period, t: Technology, op: str
+) -> Any:
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `LimitGrowthCapacityConstraint_rule`

(ANN401)